http-client/src/main/java/io/airlift/http/client/jetty/HttpRequestEvent.java

```diff
@@ -133,6 +173,16 @@ static HttpRequestEvent createHttpRequestEvent(RequestInfo requestInfo, Response
             responseCode = response.get().getStatus();
         }
 
+        long requestTotalTime = requestInfo.getRequestQueueTime() + (responseInfo.getResponseCompleteTimestamp() - requestInfo.getRequestBeginTimestamp());
```

Comment:
What's the unit for these numbers?

Reply:
I renamed these variables to clarify the units. I also found a bug which was introduced when I changed several places to use System.nanoTime() instead of System.currentTimeMillis() based on your previous comment. Will add a fixup shortly.

Reply:
That’s why I asked. I suspected there was a bug :)